src/main/java/com/chessmates/repository/MockGameRepository.groovy

```diff
@@ -14,7 +14,7 @@ class MockGameRepository implements GameRepository {
     Map store = new HashMap()
 
     @Override
-    void save(game) {
```

Comment:
It is - you can remove if you like. My comment was just that it is feasible that in the future saving a single game may make sense. I really don't feel strongly about this so consider this resolved.